packages/app-file-manager/src/modules/FileManagerRenderer/FileManagerView/configComponents/Browser/FileAction.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { CompositionScope } from "@webiny/react-composition";
+import { AcoConfig, RecordActionConfig } from "@webiny/app-aco";
+
+const { Record } = AcoConfig;
+
+export { RecordActionConfig as FileActionConfig };
+
+export const FileAction: React.FC<React.ComponentProps<typeof AcoConfig.Record.Action>> = props => {
```

Comment:
Please remove all usages of `React.FC`. We recently did a huge refactor that removes all the FC's from all the codebase. We don't want to rely on that type any more, since Facebook keeps changing waht this type really means. Simply add a type to props directly:

```tsx
type FileActionProps = React.ComponentProps<typeof AcoConfig.Record.Action>;

export const FileAction = (props: FileActionProps) => { ...}
```